src/promptflow-tools/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features Added
 - Add "detail" to "Azure OpenAI GPT-4 Turbo with Vision" and "OpenAI GPT-4V" tool inputs.
+- Address the prompt injection issue by roles escape and unescape.
```

Comment:
the change does not address prompt injection.

Reply:
What I think you do in this PR is to Avoid unintended parsing by role process to user inputs in prompt templates.

Reply:
updated